pkg/minikube/cruntime/cri.go

```diff
@@ -383,7 +399,11 @@ image-endpoint: unix://{{.Socket}}
 	if err := t.Execute(&b, opts); err != nil {
 		return err
 	}
-	return cr.Run(fmt.Sprintf("sudo mkdir -p %s && printf %%s \"%s\" | sudo tee %s", path.Dir(cPath), b.String(), cPath))
+	c := exec.Command("/bin/bash", "-c", fmt.Sprintf("sudo mkdir -p %s && printf %%s \"%s\" | sudo tee %s", path.Dir(cPath), b.String(), cPath))
+	if _, err := cr.RunCmd(c); err != nil {
+		return errors.Wrap(err, "populateCRIConfig")
```

Comment:
Like elsewhere, this code doesn't know it's own function name, so this will likely go stale and get repeated up the error chain. Just say "run" here, and let the caller of populateCRIConfig wrap it with the name it knows this code by.